sphinxcontrib/chapeldomain/chapel.py

```diff
@@ -27,6 +27,37 @@ class ChapelLexer(RegexLexer):
     aliases = ['chapel', 'chpl']
     # mimetypes = ['text/x-chapel']
 
+    known_types = ('bool', 'bytes', 'complex', 'locale', 'imag', 'int',
+                   'nothing', 'opaque', 'range', 'real', 'string', 'uint',
+                   'void')
+
+    type_modifiers = ('atomic', 'single', 'sync',
+                      'borrowed', 'owned', 'shared', 'unmanaged')
```

Comment:
This is also not alphabetized, though I do see that you're trying to group it by category.  Everything else is alphabetized, though

Reply:
Alphabetization is also good, I think. So I split this into two tuples `type_modifiers_par`, `type_modifiers_mem`